Userland/Libraries/LibTLS/TLSv12.h

```diff
@@ -132,8 +132,10 @@ enum HandshakeType {
 
 enum class HandshakeExtension : u16 {
     ServerName = 0x00,
-    ApplicationLayerProtocolNegotiation = 0x10,
+    SupportedGroups = 0x0a,
```

Comment:
With regards to the RFC's, I see that this extension is called `elliptic_curves` in [TLS1.2](https://datatracker.ietf.org/doc/html/rfc8422#section-5.1), while it is called `supported_groups` in [TLS1.3](https://datatracker.ietf.org/doc/html/rfc8446#section-4.2.7).